apps/blockchain-mailer/src/store/envStore.ts

```diff
@@ -0,0 +1,27 @@
+class EnvStore {
+
+    get isDev() {
+        return process.env.NODE_ENV === 'development';
+    }
+
+    get ethereumNodeUrl() {
+        return process.env.ETHEREUM_NODE_URL;
+    }
+
+    get factoryAddress() {
+        return process.env.FACTORY_ADDRESS;
+    }
+
+    get mailgunApiKey() {
+        return this.isDev
+            ? "key-1234123412341234"
```

Comment:
Single quotes preferred - no reason except compatibility with the rest of our projects.

Reply:
My mistake.